gateway/api_definition.go

```diff
@@ -595,9 +599,33 @@ func (a APIDefinitionLoader) replaceSecrets(in []byte) []byte {
 		}
 	}
 
+	if strings.Contains(input, prefixConsul) {
+		if err := a.replaceConsulSecrets(&input); err != nil {
+			log.WithError(err).Error("Couldn't replace consul secrets")
```

Comment:
won't the linter complain if log messages start with capital letters ?

Reply:
Nope, it is regarding error messages, logs are allowed to be capital in golang